src/main/kotlin/br/com/zup/op/events/domain/EventEntity.kt

```diff
@@ -1,4 +1,10 @@
 package br.com.zup.op.events.domain
 
-class EventEntity {
+class EventEntity(
+    val topic : String
+) {
+
+    fun isValidTopic(list : ArrayList<TopicEntiy>) {
```

Comment:
when you throw a exception, the function name should be like "validateTopic".
when you return a boolean, the function name could be that you defined.